openhands/runtime/utils/edit.py

```diff
@@ -214,8 +213,9 @@ def edit(self, action: FileEditAction) -> Observation:
             if isinstance(obs, ErrorObservation):
                 return obs
             if not isinstance(obs, FileWriteObservation):
-                return FatalErrorObservation(
-                    f'Fatal Runtime in editing: Expected FileWriteObservation, got {type(obs)}: {str(obs)}'
+                return ErrorObservation(
```

Comment:
**Suggestion:** Use consistent naming for error handling to improve readability.

**Existing Code:**
```Python
return ErrorObservation(
    f'Fatal Runtime in editing: Expected FileWriteObservation, got {type(obs)}: {str(obs)}',
)
```

**Improved Code:**
```Python
return ErrorObservation(
    f'Error in editing: Expected FileWriteObservation, got {type(obs)}: {str(obs)}',
)
```

**Details:**
Consider using a consistent naming convention for error handling to improve readability and maintainability.